daemon/stats/collector.go

```diff
@@ -114,7 +115,7 @@ func (s *Collector) Run() {
 		}
 
 		for _, pair := range pairs {
-			stats, err := s.supervisor.GetContainerStats(pair.container)
+			stats, err := s.supervisor.GetContainerStats(context.Background(), pair.container)
```

Comment:
Should this be a `context.TODO()`? Mostly looking that each of these gets their own context and/or if contexts should be reused here 🤔